config/config.hpp.in

```diff
@@ -104,6 +104,7 @@
 
 // Enable MFEM features based on the Ginkgo library.
 // #define MFEM_USE_GINKGO
+// #define MFEM_GINKGO_WITH_OMP @MFEM_GINKGO_WITH_OMP@
```

Comment:
Maybe `MFEM_GINKO_USE_OPENMP` instead of `*_WITH_OMP` to be consistent with the definitions elsewhere? Would apply to `CMakeLists.txt` as well as the other GNU make related files.